### PR TITLE
Zoomed description of a product

### PR DESCRIPTION
**Pull request by Megahand**

### CODE REVIEW

**Comment by Megahand** [comment]
[rockplusinc.com Email de Mike : Zoomed description of a product](https://trello.com/c/N3chAI5I/7-rockplusinc-com-email-de-mike-zoomed-description-of-a-product)

File: src/styles/_item-details.scss [hunk 1]
```diff
@@ -93,4 +94,16 @@ img.backorder {
...
     }
   }
+  .info{
+    position: absolute;
+    right: 10px;
+    margin: 0 10px;
+    padding: 0 5px 10px 5px;
+    width: 45%;
+    text-align: left;
+
```
Comment (zeachco) [inline src/styles/_item-details.scss:107]: might be good to add `font-weight: 700; color: black; text-shadow 0 0 1px white` for readability

File: src/components/Item/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const Item = props => {
         }}></div>
         <span>
           <small>
-            <Details imgClassName={"png_alpha"} src={imgFull || NoImageAvail} thumbsSrc={imgThumb || NoImageAvail} title={name}/>
+            <Details imgClassName={"png_alpha"} src={imgFull || NoImageAvail} thumbsSrc={imgThumb || NoImageAvail} title={name} {...props}/>
```
Comment (zeachco) [inline src/components/Item/index.js]: `imgClassName={"png_alpha"}` can be changed to `imgClassName="png_alpha"` (no evaluation)

File: src/components/Item/Details/index.js [hunk 3]
```diff
@@ -56,6 +58,14 @@ class LightBox extends React.Component {
             ? thumbsSrc
             : null} alt={title} onClick={this.open.bind(this)}></img>
           <SkyLight hideOnOverlayClicked ref="customDialog" title={title}>
+          <div className={'info'}>
```
Comment (zeachco) [inline src/components/Item/Details/index.js]: `<div className={'info'}>` can be changed to `<div className="info">`

**Review by zeachco — CHANGES_REQUESTED** [review]
small modification before merge
also if you can include a small image of the changes in the description
that would be awesome :smile:

**Review by zeachco — APPROVED** [review]

**Comment by zeachco** [comment]
@Megahand I've pushed some changes addressing my comments

**Pull request merged**